src/index.js

```diff
@@ -3,20 +3,29 @@ const CyclicCollection = require('./cy_db_collection')
 const CyclicIndex = require('./cy_db_index')
 const CyclicItem = require('./cy_db_item')
 
-class CyclicDb {
- 
-    static item(collection,key){
-        return new CyclicItem(collection, key)
+class CyclicDb extends Function{
+    constructor() {
+        super('...args', 'return this._bound._call(...args)')
+        this._bound = this.bind(this)
+        return this._bound
+      }
+      
+    _call(table_name) {
+        process.env.CYCLIC_DB = table_name
+        return new CyclicDb
     }
 
-    static collection(collection){
+    item(collection,key){
+        return new CyclicItem(collection, key)
+    }
+ 
+    collection(collection){
         return new CyclicCollection(collection)
     }
 
-    static index(name){
+    index(name){
         return new CyclicIndex(name)
     }
 }
 
-
-module.exports = CyclicDb
\ No newline at end of file
+module.exports = new CyclicDb
```

Comment:
This means that each file that imports/requires this will get its own instance of the DB class. Not sure that it creates problems but feels off.